### DISCUSSION TITLE
P1 Camera HomeBridge

### DISCUSSION THREAD

**Discussion by honzis1364** [greghesp/ha-bambulab#527] [Q&A]
Hello, after 8 hours of trying, I'm giving up and seeking for your help. 

Has anyone managed to add P1 Camera into HomeBridge? I don't have a problem viewing it in HA (all tho it's under "Sensors"). But I can't get it into the Home Bridge. I suspect that it has something to do with format (X1 is probably H.264, which is acceptable, P1 has image stream if I'm not mistaken). 
I've tried using the Scrypted add-on with FFmpeg plugin, but I dont know what to fill into the Input fields to set it up.

**Comment by AdrianGarside** [comment]
The P1 camera is no longer just an image that updates every ~2 seconds but is now exposed as an actual camera. But since it's still driven by the same image every couple of seconds it may not be any easier for you to consume.
